### DISCUSSION TITLE
Positioning Map Controls such as Jump to my location or Driving directions button

### DISCUSSION THREAD

**Discussion by dcesca** [react-native-maps/react-native-maps#4345] [Q&A] [answered]
### Summary

Hi, I think this picture explains in a second my issue:
![image](https://user-images.githubusercontent.com/5036773/180423495-84c8177e-610a-49ae-81f3-12fb319d26d4.png)

I can't find a way to position the "map controls"; they appear under my ui components and I'd need to move them around a bit.

Is it possible?

### Reproducible sample code

```TSX
<MapView
          ref={_map}
          showsCompass={true}
          showsPointsOfInterest={false}
          onRegionChangeComplete={onRegionChange}
          showsMyLocationButton={true}
          showsUserLocation={true}
          initialRegion={mapRegion}
          style={styles.container}
          provider={PROVIDER_GOOGLE}
          customMapStyle={mapStandardStyle}>
<MapView.Marker
                  key={index}
                  coordinate={marker.coordinate}
                  onPress={e => onMarkerPress(e)}
                  image={require('../assets/images/map_marker.png')}
              >

              </MapView.Marker>
</MapView>
```

### Steps to reproduce

Create a Map with Google Maps provider and with showsMyLocationButton={true} and  showsUserLocation={true}
Add a marker and touch it so the directions button appear (on Android) in the bottom right corner.
The myPositionButton appears (on android) in the top right corner)

### Expected result

I expect to be able to control in someway the position of the map "default controls"

### Actual result

Map controls position themselves in their default positions and I cannot move them where I want

### React Native Maps Version

0.30.2

### What platforms are you seeing the problem on?

Android, iOS (Google Maps)

### React Native Version

0.62.2

### What version of Expo are you using?

SDK 45

### Device(s)

Any Android I tried (recent devices)

**Comment by monholm** [comment, marked as answer]
Hi @dcesca.

Check out the `mapPadding` prop in the [docs](https://github.com/react-native-maps/react-native-maps/blob/master/docs/mapview.md) :)
